stories/list-grid/list-grid.stories.js

```diff
@@ -0,0 +1,1880 @@
+                        <div class="grid-list-item-body-example">
+                            <span class="fd-avatar fd-avatar--s grid-list-item-body-example-avatar" role="presentation" aria-label="John Doe"></span>
+                            <div>
+                                <h4 class="fd-title fd-title--h4">John Li</h4>
+                                <p>Product Owner, Company B</p>
+
+                                <div>
+                                    <p>781 Main Street</p>
+                                    <p>Anytown, SD 57401</p>
+                                    <p>USA</p>
+                                </div>
+
+                                <a href="#" class="fd-link" tabindex="0">john_li@example.com</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+singleSelectLeftMode.storyName = 'Single select left mode';
+
+singleSelectLeftMode.parameters = {
+    docs: {
+        storyDescription: `
+One item in the grid list can be selected. For this, the grid list provides radio buttons on the left side of each item toolbar.
+To turn on the Single select left mode, add the \`fd-grid-list__mode--single-select-left\` modifier class to the main element.
+`
+    }
+};
+
+export const multiSelectMode = () => `<div class="fd-grid-list fd-grid-list__mode--multi-select">
+    <div class="fd-container">
+        <div class="fd-row">
+            <div class="fd-col fd-col--12">
+                <div class="fd-toolbar fd-toolbar--solid">
+                    <span>Products (420)</span>
+
+                    <span class="fd-toolbar__spacer"></span>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Sort">
+                        <i class="sap-icon--sort"></i>
+                    </button>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Filter">
+                        <i class="sap-icon--filter"></i>
+                    </button>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Group">
+                        <i class="sap-icon--group-2"></i>
+                    </button>
+                </div>
+            </div>
+        </div>
+
+        <div class="fd-row">
+            <div  class="fd-col fd-col--12 fd-col-md--6 fd-col-lg--4 fd-col-xl--3">
+                <div tabindex="0" class="fd-grid-list__item fd-grid-list__item--link is-selected">
+                    <div class="fd-toolbar fd-grid-list__toolbar">
+                        <input type="checkbox" class="fd-checkbox fd-checkbox--compact fd-grid-list__checkbox--input" id="Ai4ez611c" checked>
+                        <label class="fd-checkbox__label fd-checkbox__label--compact fd-grid-list__checkbox--label" for="Ai4ez611c"></label>
```

Comment:
Same feedback as for the radio buttons, except for some reason the content of the grid item doesn't read out when you access this checkbox, just 'checkbox'. Every input needs some sort of label.

Reply:
fixed